scripts/clean_validate.py

```diff
@@ -0,0 +1,41 @@
+# clean_validate.py
+# author: Weilin Han
+# date: 2024-10-20
+
+import click
+import os
+import sys
+import pandas as pd
+import pandera as pa
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.clean_data import extract_column_name, read_raw_data, clean_data, write_data
+from src.validate_data import build_schema_from_csv
+
+@click.command()
+@click.option('--raw-data-file', type=str, help="Path to raw data file")
+@click.option('--name-file', type=str, help="Path to dirctory where names file resides")
+@click.option('--write-to', type=str, help="Path to directory where cleaned data will be written to")
+
+def main(raw_data_file, name_file, write_to):
+    """Clean raw data and validate it."""
+    # Extract column names from .names file
+    colnames = extract_column_name(name_file)
+
+    # Read raw data
+    imported_data = read_raw_data(raw_data_file, colnames)
+
+    # Removing id column and relabel diagnosis column
+    cleaned_data = clean_data(imported_data)
+
+    # Validate cleaned data
+    # Load the CSV config file
+    data_config_file = '/data/processed/data_config.csv'
```

Comment:
`data_config_file` should be a command line argument, its value should be the path to this file.